ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -40,14 +40,13 @@ StatusDialog {
         })
 
         function fetchSuggestedRoutes() {
```

Comment:
Here's a proposal for some future improvements in the `SwapModal`. I'd open a task if it makes sense to you as well:
IMO the `SwapModal` should be responsible only in updating the form. In this case it should not bear the logic for route fetching. This means that the `SwapModal` can modify the flow state only my calling functions on the input swap form.

cc @Khushboo-dev-cpp

Reply:
https://github.com/status-im/status-desktop/issues/16648